cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -301,6 +304,52 @@ describe(["@tier2"], "Application inventory filter validations", function () {
         application.delete();
         application1.delete();
     });
+    it("Archetype filter validations", function () {
+        const tags = createMultipleTags(2);
+
+        //validate archetype1 is shown in applications archetype filter and application1 is present
+        const archetype1 = new Archetype(
```

Comment:
Same as we spoke before, please implement a list, array, map, or anything you need to make it work, but do something iterable, not hardcoded thing like this.

Reply:
OK, following our conversation, in this particular case we can avoid this approach. Resolving this...